metadata.rb

```diff
@@ -0,0 +1,19 @@
+# Encoding: utf-8
+
+name             'cerner_splunk_ingredient'
+maintainer       'Cerner Innovation, Inc.'
+maintainer_email 'splunk@cerner.com'
+license          'Apache 2.0'
+description      'Installs Splunk'
+long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+version          '1.0.0'
```

Comment:
If we're going to be making changes parallel to cerner_splunk, 0.1.0 is a better choice.